staff/carlos-luis-querecuto/skylab-semantics/README.md

```diff
@@ -0,0 +1,1291 @@
+# Skylab semantics
```

Comment:
@carlosq09 this file does not belong to this branch. Please remove it and confirm when it's done. Thanks.

Reply:
Fixed!